.github/workflows/gh-pages.yml

```diff
@@ -10,8 +9,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Build & Deploy to GitHub Pages
+      - uses: ruby/setup-ruby@v1
         with:
-          REMOTE_BRANCH: gh-pages
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        uses: yurikoval/middleman-gh-pages-action@master
+          ruby-version: 3.1.2
```

Comment:
Doesn't this break anything else since it was always mentioned to use 2.6.9?

Reply:
At some point we need to upgrade, we can't stay on old versions forever. The newer Ruby version is what caused the issue with the API table but that's now resolved.

3.1 isn't the latest Ruby version but recent enough that it'll be supported for a while (march 2025).